runelite-client/src/main/java/net/runelite/client/plugins/regenmeter/RegenMeterPlugin.java

```diff
@@ -140,6 +148,7 @@ private void onTick(GameTick event)
 
 		int currentHP = client.getBoostedSkillLevel(Skill.HITPOINTS);
 		int maxHP = client.getRealSkillLevel(Skill.HITPOINTS);
+		checkHealthPercentage = ((Math.round(hitpointsPercentage * 100)) == config.regenWarning() || (Math.round(hitpointsPercentage * 100)) == config.regenWarning() - 1);
```

Comment:
reuse Math.round(hitpointsPercentage * 100). Also why the check for config.regenWarning() - 1?

Reply:
Because when using rapid heal it can be on a different tick.